active/community.md

```diff
@@ -0,0 +1,75 @@
+# WG Charter Template
+
+## Scope of responsibilities
+
+This group exists to support the official community platforms and those in elevated roles on those platforms.
+
+The Forum and the Discord are core parts of the overall community and those that help and moderate these platforms take significant time from their day to help others. We can support them through training and taking action to make their lives easier.
+
+The broad goals of the group are:
+  - facilitate and centralise discussion between moderators and admins on the various platforms
```

Comment:
I wonder how much input you can get from people (e.g. me) that are admins on the Discord while they're not in the group themselves. But I imagine everyone is happy to share, and ideally some group members would have reasonable access levels to at least the mod only sections for example.

Reply:
Agreed that not all admins/moderators need to be a part of the group, but I would prefer representation and input from those groups.